_sql/hotelify.sql

```diff
@@ -318,7 +369,21 @@ INSERT INTO `reservation_room` (`reservation_id`, `room_id`, `checkin_date`, `ch
 (20, 35, '2017-09-18', '2017-09-19'),
 (21, 53, '2016-11-07', '2016-11-22'),
 (22, 55, '2015-01-01', '2015-01-03'),
-(23, 60, '2015-05-01', '2015-06-01');
```

Comment:
we need to seed these reservations in a way that eventually we would be able to demo our quick book feature well by using these data. for example, if we want a demo user to quick book hotel gage, **Ideally**, we want to show TAs three cases:
- search results return walter gage when we requested for 1 room from date x to date y
- search results does not return walter gage when we requested for z rooms from date x to date y 
- search results does not return walter gage when we requested for 10000 rooms from date 2020-12-01 to 2020-12-05(a date range that has no reservations)

It would be great if we can show them all three cases together since that is our core business logic.